projects/cashmere/src/lib/pop/popover.component.scss

```diff
@@ -0,0 +1,135 @@
+@import '~@angular/cdk/_overlay';
```

Comment:
It is a little hard to see the arrow and top of the popup, especially with our standard bg. Maybe a border? This is similar styling as the last popover but I wanted to point it out. Not a huge deal

![image](https://user-images.githubusercontent.com/1934453/56062572-13943380-5d2a-11e9-9e80-ea4bcc8d86eb.png)